infra/app.py

```diff
@@ -23,6 +24,14 @@
     env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
 )
 
+data = DataStack(
```

Comment:
"data" in a name is IMO problematic, because the word means nothing in isolation. In this case it's extra confusing because it overlaps with the name of the application. I would suggest a name like "storage".

Reply:
I had `storage` first but then I renamed it to `data`. Renamed back to `storage` in af1b8e9eabae7ddb53e2c401e883851e06e8a9b3